src/main/java/edu/kit/ipd/crowdcontrol/objectservice/rest/resources/ExperimentResource.java

```diff
@@ -0,0 +1,427 @@
+        AlgorithmTaskChooserRecord taskChooserRecord = getOrThrow(
+                algorithmsOperations.getTaskChooser(experimentRecord.getAlgorithmTaskChooser())
+        );
+        Map<AlgorithmTaskChooserParamRecord, String> taskChooserParams = algorithmsOperations.getTaskChooserParams(
+                experimentRecord.getAlgorithmTaskChooser(), experimentRecord.getIdExperiment());
+        AlgorithmAnswerQualityRecord answerQualityRecord = getOrThrow(
+                algorithmsOperations.getAnswerQualityRecord(experimentRecord.getAlgorithmQualityAnswer())
+        );
+        Map<AlgorithmAnswerQualityParamRecord, String> answerQualityParams = algorithmsOperations.getAnswerQualityParams(
+                experimentRecord.getAlgorithmQualityAnswer(), experimentRecord.getIdExperiment());
+        AlgorithmRatingQualityRecord ratingQualityRecord = getOrThrow(
+                algorithmsOperations.getRatingQualityRecord(experimentRecord.getAlgorithmQualityRating())
+        );
+        Map<AlgorithmRatingQualityParamRecord, String> ratingQualityParams =
+                algorithmsOperations.getRatingQualityParams(experimentRecord.getAlgorithmQualityRating(), experimentRecord.getIdExperiment());
+
+        return ExperimentTransform.toProto(experimentRecord,
+                state,
+                constraintRecords,
+                platforms,
+                tagRecords,
+                taskChooserRecord,
+                taskChooserParams,
+                answerQualityRecord,
+                answerQualityParams,
+                ratingQualityRecord,
+                ratingQualityParams);
+    }
+
+    /**
+     * Returns a experiment which was specified by :id
+     * @param request  Request provided by Spark.
+     * @param response Response provided by Spark.
+     * @return The experiment if it was found
+     */
+    public Experiment get(Request request, Response response) {
+        int id = getParamInt(request, "id");
+
+        return fetchExperiment(id);
+    }
+
+    /**
+     * Will take the id of an experiment and return the platform tree with
+     * all published platforms with the according calibrations
+     * @param id the id of the experiment
+     * @return returns a list of populations with a platform
+     */
+    private List<Experiment.PlatformCalibrations> getPlatforms(int id) {
+
+        Function<ExperimentsCalibrationRecord, Calibration> toCalibration = record -> {
+            CalibrationAnswerOptionRecord a = getOrThrow(
+                    calibrationOperations.getCalibrationAnswerOption(record.getAnswer())
+            );
+            return getOrThrow(calibrationOperations.getCalibration(a.getCalibration()));
+        };
+
+        Function<Map.Entry<String, List<Calibration>>, Experiment.PlatformCalibrations> toPopulation = entry ->
+                Experiment.PlatformCalibrations.newBuilder()
+                //.setPlatformId(entry.getKey)
```

Comment:
no it was commented out before - I had to wait with this until the new protobuf files where in, and forgot about it. You can comment it in. Not on keyboard right now.